main.py

```diff
@@ -1,19 +1,30 @@
+# -*- encoding: utf-8 -*-
 import argparse
 from converter.converter import *
 
-
 CELSIUM_INPUT = ['celsius', 'cels', 'cel', '-c', 'c']
 FAH_INPUT = ['fahrenheit', 'fahren', 'far', '-f', 'f']
 QUIT_INPUT = ['quit', 'q', 'exit']
 
-
 parser = argparse.ArgumentParser()
 parser.add_argument("-v", "--verbose", help="It just return your argument")
 parser.add_argument("-f", "--fahrenheit", help="Take temperature in Fahrenheit and convert to Celsius", type=float)
 parser.add_argument("-c", "--celsius", help="Take temperature in Celsius and convert to Fahrenheit", type=float)
 
 args = parser.parse_args()
 
+
+def check_number(arg):
```

Comment:
@MariaMaximova, I would suggest renaming it to `is_float` as it checks if a string contains a float number. `check_number` name does not represent this meaning therefore is not acceptable here, I think.

Reply:
Resolved in this [commit](https://github.com/MariaMaximova/convert_temp/pull/9/commits/ea07c0df2d274ab51fb7e4346d05465b52c56248)